data_validation/cli_tools.py

```diff
@@ -141,7 +141,8 @@
 def get_parsed_args():
     """Return ArgParser with configured CLI arguments."""
     parser = configure_arg_parser()
-    return parser.parse_args()
+    args = None if sys.argv[1:] else ["--help"]
```

Comment:
This breaks the code after testing with `data-validation`. I'm not understanding the logic here...

We want to do something similar to [this](https://stackoverflow.com/questions/4042452/display-help-message-with-python-argparse-when-script-is-called-without-any-argu) where if no arguments are provided, we print the help menu.

Reply:
The logic is when you go to `parse_args()` function it has a args parameter which is default to None, so earlier we were calling parse_args() with empty parameter, in which args parameter will be default to None. Now `sys.argv[1:]` returns a list of argument passed with data-validation command, so if we pass any argument it will work with our original logic, but if we don't pass any argument, `sys.argv[1:]` will return null and we will pass the --help flag in args parameter.

Reply:
Makes sense. Only suggestion is to check the length instead of list slicing for efficiency. ie `if len(sys.argv) == 1`

Reply:
Sure, fixed it with this [commit](https://github.com/GoogleCloudPlatform/professional-services-data-validator/pull/627/commits/dd615958b1e10101240bda629cf538ffca5ef9f9).